### ISSUE TITLE
Follow up issues in css-text/i18n/*/css-text-line-break-* tests

### ISSUE THREAD

**Issue by kojiishi** [w3c/web-platform-tests#11161]
Because the PR #10860 from @xfq and @r12a touches over 500 files and due to limited bandwidth, the discussion in PR went to land once and work continuously in following PRs. This issue is to track review feedback in the PR.

@gsnedders [comments](https://github.com/w3c/web-platform-tests/pull/10860#pullrequestreview-118092625):
- [ ] In the 3 loops it has, the 2nd one can be merged.

@mrego [comments](https://github.com/w3c/web-platform-tests/pull/10860#pullrequestreview-122528144):
- [ ] Indentation code style preferences vary by people.
- [ ] Not necessary to split style elements.
- [ ] String data v.s. json preference.
- [ ] The `name` variable is not used.
- [ ] Code style: omit semicolon or not.
- [ ] Put `<div id=log>` directly in HTML.
- [ ] Wrap long lines.
- [ ] `document.querySelector('body')` can be `document.body`.

@frivoal [comments](https://github.com/w3c/web-platform-tests/pull/10860#pullrequestreview-122852115):
- [x] Comments at the bottom is misleading.
- [ ] `-cj-` tests are missing for zh.
- [ ] `-in-` tests are missing for other languages than ja and zh.
- [ ] `-cpm-` and `-po-` for other than ja and zh.
- [ ] Same remark for `-hyphens` tests.
- [ ] Add `-pr-` tests.

**Comment by r12a** [comment]
@frivoal for 

> -cj- tests are missing for zh, arbitrary language other than ja or zh, and lang unset.

help me understand why the extra tests are needed. Characters with the CJ property are only Japanese (see https://unicode.org/cldr/utility/list-unicodeset.jsp?a=[:Line_Break=Conditional_Japanese_Starter:]) and there's no language-specific behaviour mentioned text in the CSS spec (see https://drafts.csswg.org/css-text-3/#line-break-property) for these characters.

>  The following breaks are forbidden in strict line breaking and allowed in normal and loose:
> breaks before Japanese small kana or the Katakana-Hiragana prolonged sound mark: i.e. characters with the Unicode Line Break property CJ. (See LineBreak.txt in [UNICODE].) 

Are you asking for additional tests to show that the behaviour described in the assertion occurs whatever the language of the text?

**Comment by r12a** [comment]
Perhaps an easy way to answer the above question is to provide an assertion for such a test.  The assertion for the current CJ tests read similar to:

> With line-break:strict, a browser will NOT allow a conditional Japanese starter at the beginning of a line.

**Comment by frivoal** [comment]
> Are you asking for additional tests to show that the behaviour described in the assertion occurs whatever the language of the text?

Yes. Since the spec was deliberate in tying some of the behaviors to the language being declared, but some other to occur regardless of the language declaration. My understanding is that for the characters that may have different line breaking behaviors in different languages, declaring the language is required so that they get processed right. For those that cannot, then it should work even if the author has forgotten to declare the language, either on the whole document, in which case language is not set, or on a short piece of CJK text inside something else which is language tagged, in which case the lang may be wrong.

**Comment by frivoal** [comment]
All corrections that relate to correctness or coverage have been done. The rest is a matter of coding style. Since nobody seems interested in fixing these, I suggest closing.
